commonmark-spannable/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
Any reason why we are not updating this together with targetsdk?

Reply:
I can do that as a later PR, but the dependency updates required a `compileSdkVersion` of 30. I didn't want to do too many changes at once in case it introduces regressions.

Reply:
It was `androidx.lifecycle` that required the bump, FYI